addons/project_timesheet_holidays/data/holiday_timesheets_demo.xml

```diff
@@ -1,97 +1,14 @@
-            <field name="task_id" model="project.task" eval="obj().env.ref('hr_holidays.hr_holidays_cl_qdp').holiday_status_id.timesheet_task_id.id"/>
-            <field name="holiday_id" ref="hr_holidays.hr_holidays_cl_qdp"/>
-        </record>
-
-        <record id="hr_holidays_analytic_line_7" model="account.analytic.line">
-            <field name="name">Sick Day</field>
-            <field name="user_id" ref="base.user_demo"/>
-            <field name="date" eval="(datetime.now() + relativedelta(day=1, weekday=0)).strftime('%Y-%m-%d 01:00:00')"/>
-            <field name="unit_amount">8.00</field>
-            <field name="project_id" model="project.project" eval="obj().env.ref('hr_holidays.hr_holidays_sl_qdp').holiday_status_id.timesheet_project_id.id"/>
-            <field name="task_id" model="project.task" eval="obj().env.ref('hr_holidays.hr_holidays_sl_qdp').holiday_status_id.timesheet_task_id.id"/>
-            <field name="holiday_id" ref="hr_holidays.hr_holidays_sl_qdp"/>
-        </record>
-
-        <record id="hr_holidays_analytic_line_8" model="account.analytic.line">
-            <field name="name">Sick Day</field>
-            <field name="user_id" ref="base.user_demo"/>
-            <field name="date" eval="(datetime.now() + relativedelta(day=1, weekday=0) + relativedelta(days=1)).strftime('%Y-%m-%d 23:00:00')"/>
-            <field name="unit_amount">8.00</field>
-            <field name="project_id" model="project.project" eval="obj().env.ref('hr_holidays.hr_holidays_sl_qdp').holiday_status_id.timesheet_project_id.id"/>
-            <field name="task_id" model="project.task" eval="obj().env.ref('hr_holidays.hr_holidays_sl_qdp').holiday_status_id.timesheet_task_id.id"/>
-            <field name="holiday_id" ref="hr_holidays.hr_holidays_sl_qdp"/>
-        </record>
-
-        <record id="hr_holidays_analytic_line_9" model="account.analytic.line">
-            <field name="name">Sick Day</field>
-            <field name="user_id" ref="base.user_demo"/>
-            <field name="date" eval="(datetime.now() + relativedelta(day=1, weekday=0) + relativedelta(days=2)).strftime('%Y-%m-%d 23:00:00')"/>
-            <field name="unit_amount">8.00</field>
-            <field name="project_id" model="project.project" eval="obj().env.ref('hr_holidays.hr_holidays_sl_qdp').holiday_status_id.timesheet_project_id.id"/>
-            <field name="task_id" model="project.task" eval="obj().env.ref('hr_holidays.hr_holidays_sl_qdp').holiday_status_id.timesheet_task_id.id"/>
-            <field name="holiday_id" ref="hr_holidays.hr_holidays_sl_qdp"/>
-        </record>
-
+    <data>
+        <function model="hr.leave" name="_remove_resource_leave">
+            <value eval="[ref('hr_holidays.hr_holidays_sl'), ref('hr_holidays.hr_holidays_cl_qdp'), ref('hr_holidays.hr_holidays_sl_qdp')]"/>
+        </function>
```

Comment:
Why do you need to remove the `resouce.calendar.leave` for those leaves and after `_timesheet_create_lines` you recreate those ones? 🤔

Reply:
Oh okay, I saw your comment above for Vincent. 🙂